### ISSUE TITLE
Warn a user about receiving WETH/WXDAI tokens after cancellation

### ISSUE THREAD

**Issue by elena-zh** [gnosis/ido-ux#672]
Discussed  in  #669  https://github.com/gnosis/ido-ux/pull/669#issuecomment-858453286

When a user cancels an order in ETH, he/she receives back WETH tokens in Mainnet/Rinkeby.
When a user cancels an order in XDAI, he/she receives back WXDAI tokens in xDAI.

It would be nice to warn a user somehow about it.  As an example, add a message to the cancellation confirmation pop-up or so.

**Comment by alongoni** [comment]
Currently when a user wants to cancel an order, a modal like this appears:
![GA-CancelOrder](https://user-images.githubusercontent.com/622217/123313590-0a08d780-d500-11eb-8aed-7942291cc938.png)
I think It's **too invasive** and doesn't provide too much information.

So I suggest this design that include information to review about the order that the user is cancelling. Also we add the information about receiving WETH/XDAI tokens after cancellation.
![12-B- GnosisAuction - Cancelling](https://user-images.githubusercontent.com/622217/123313639-14c36c80-d500-11eb-821c-f2bc44310ac9.png)

What do you think? @josojo @cmagan

**Comment by elena-zh** [comment]
IMO, I do not think that we need the same info as in the place order confirmation modal.
I mean, It will be enough to have 2 lines with info:
 - < token > tokens sold
 - Min (max) price per < selling token >

I think there is no need in 'Min token received' field as more likely users will cancel orders in case when a price is below a current price, and they will receive nothing when an auction ends. 

Thanks

**Comment by alongoni** [comment]
Thanks @elena-zh for your comments! :)
I've upgrade the design, please let me know if you think it will work better now.
![12-B- GnosisAuction - Cancelling](https://user-images.githubusercontent.com/622217/123423623-2d7e6180-d596-11eb-8c5e-41fe19f453f1.png)

**Comment by elena-zh** [comment]
@alongoni , cool, thank you!
Let wait for  @josojo and @cmagan thoughts about it.

**Comment by josojo** [comment]
Yes. looks really good! I think it would be nice to make somehow clear that this is a confirmation window. maybe we can call it: Confirm order Cancelation. 
Though this is quite long.

If you don't have a good suggestion, then I think we can keep it as it is

**Comment by josojo** [comment]
@maria-vslvn 

This has a higher prio than the order table improvements, in case you have capacity :)

**Comment by alongoni** [comment]
I think the length of the title is ok!!
[12-B- GnosisAuction - Cancelling](https://user-images.githubusercontent.com/622217/123689182-52d6ce00-d829-11eb-9215-f917d2ef6ec7.png)

**Comment by cmagan** [comment]
looks good!
Just making sure you're aware that the warning "You'll receive back WXDAI tokens instead of native xDAI"
Is only relevant in case the bidding token is the the wrapped version of the native token (WXDAI / WETH) 
I think it'd be good to display this warning only if this is actually the case

**Comment by josojo** [comment]
Yes, I also like it. Looks good

**Comment by elena-zh** [comment]
@maria-vslvn , an additional message in the Cancel pop-up should be displayed in the next cases: 
1. Mainnet/Rinkeby network:  Bidding token is ETH
2. xDAI network: Bidding token is XDAI

The message should not be displayed when bidding tokens are not in the list of mentioned above. 

Label of the field 'Max (min) < bidding token >paid per < selling token >' should show an appropriate value when:
 -  prices are 'ordinary':  'Max  < bidding token >paid per < selling token >'
 - prices are inverted:  'Min< bidding token >paid per < selling token >'

You can copy this logic from the 'Confirm order' modal
![image](https://user-images.githubusercontent.com/70885163/124950524-27b76000-e01b-11eb-90df-31679bab84c4.png)

**Comment by elena-zh** [comment]
@maria-vslvn , there the auction you can play with
https://ido-ux.dev.gnosisdev.com/#/auction?auctionId=286&chainId=4#topAnchor

1. connect a wallet in Rinkeby, 
2. place an order there, 
3. then press on the Cancel button in the 'Your Orders' section to call the cancellation pop-up

For any other token pair, you can use any ongoing auction in the app.
